packages/core/breadcrumb.js

```diff
@@ -1,19 +1,19 @@
 const { isoDate } = require('./lib/es-utils')
 
 class BugsnagBreadcrumb {
-  constructor (name = '[anonymous]', metaData = {}, type = 'manual', timestamp = isoDate()) {
+  constructor (message, metadata, type, timestamp = isoDate()) {
```

Comment:
These default values have been removed in favour of the validation/default values being done in the `client.leaveBreadcrumb()` method rather than being split between there and here.